apps/dashboard/src/components/primitives/control-input/variable-plugin/index.ts

```diff
@@ -29,7 +29,7 @@ export function createVariableExtension({ viewRef, lastCompletionRef, onSelect }
   );
 }
 
-export const VARIABLE_REGEX = /{{([^{}]+)}}/g;
```

Comment:
changed to be a string instead of object as it's reused across and `exec` doesn't work from first time if was called before